app/services/ccms/requestors/case_add_requestor.rb

```diff
@@ -84,209 +73,216 @@ def request_xml
-        xml.__send__('ns2:OtherParty') do
-          xml.__send__('ns2:OtherPartyID', "OPPONENT_#{opponent.generate_ccms_opponent_id}")
-          xml.__send__('ns2:SharedInd', false)
-          xml.__send__('ns2:OtherPartyDetail') do
-            xml.__send__('ns2:Person') do
-              xml.__send__('ns2:Name') do
-                xml.__send__('ns2:Title', '.')
-                xml.__send__('ns2:Surname', last_name)
-                xml.__send__('ns2:FirstName', first_name)
-              end
-              xml.__send__('ns2:Address')
-              xml.__send__('ns2:RelationToClient', 'NONE')
-              xml.__send__('ns2:RelationToCase', 'OPP')
-              xml.__send__('ns2:ContactDetails')
-            end
-          end
-        end
-      end
+      # Commented out until CCMS :title issue is resolved
+      # def generate_opponent(xml, opponent) # rubocop:disable Metrics/MethodLength
+      #   first_name, last_name = opponent.split_full_name
+      #   xml.__send__('casebio:OtherParty') do
+      #     xml.__send__('casebio:OtherPartyID', "OPPONENT_#{opponent.generate_ccms_opponent_id}")
+      #     xml.__send__('casebio:SharedInd', false)
+      #     xml.__send__('casebio:OtherPartyDetail') do
+      #       xml.__send__('casebio:Person') do
+      #         xml.__send__('casebio:Name') do
+      #           xml.__send__('common:Title', ' ')
+      #           xml.__send__('common:Surname', last_name)
+      #           xml.__send__('common:FirstName', first_name)
+      #         end
+      #         xml.__send__('casebio:Address')
+      #         xml.__send__('casebio:RelationToClient', 'NONE')
+      #         xml.__send__('casebio:RelationToCase', 'OPP')
+      #         xml.__send__('casebio:ContactDetails')
+      #       end
+      #     end
+      #   end
+      # end
 
       def generate_involved_child(xml, child) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
         first_name, last_name = child.split_full_name
-        xml.__send__('ns2:OtherParty') do
-          xml.__send__('ns2:OtherPartyID', "OPPONENT_#{child.generate_ccms_opponent_id}")
-          xml.__send__('ns2:SharedInd', false)
-          xml.__send__('ns2:OtherPartyDetail') do
-            xml.__send__('ns2:Person') do
-              xml.__send__('ns2:Name') do
-                xml.__send__('ns2:Title', '.')
-                xml.__send__('ns2:Surname', last_name)
-                xml.__send__('ns2:FirstName', first_name)
+        xml.__send__('casebio:OtherParty') do
+          xml.__send__('casebio:OtherPartyID', "OPPONENT_#{child.generate_ccms_opponent_id}")
+          xml.__send__('casebio:SharedInd', false)
+          xml.__send__('casebio:OtherPartyDetail') do
+            xml.__send__('casebio:Person') do
+              xml.__send__('casebio:Name') do
+                xml.__send__('common:Title', 'MASTER')
+                xml.__send__('common:Surname', last_name)
+                xml.__send__('common:FirstName', first_name)
```

Comment:
Should these lines use the `casebio` namespace instead of `common`? In the original code they were all `ns2` which suggests they should all map to the same new namespace (`casebio`).

Reply:
I have changed this, it looks like it has been set as that since the first commit, but you are correct that it has always been `ns2/casebio `as per the others fields in this attribute. I will make sure that we get it checked on CCMS submission to see if there is any weird behaviour as the result of the change.